samples/drivers/led_ws2812/boards/nrf7002dk_nrf5340.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_SPI=n
```

Comment:
Bit confused by this as there is no such board in zephyr?

Reply:
By default SPI is turned on (https://github.com/zephyrproject-rtos/zephyr/blob/95a5f5178fed90dc193c5c934f4a61559f203cf8/samples/drivers/led_ws2812/Kconfig#L7) but since I used I2S, we don't need it.

Reply:
The comment was regarding the name of the overlay file, as there is no such board as `nrf7002dk_nrf5340` in Zephyr upstream since it's part of nRF Connect SDK.

Reply:
I see. Thank you for clarifying. I didn't know it was not part of the upstream (yet). I see `nrf5340dk_nrf5340` is already here (https://github.com/zephyrproject-rtos/zephyr/tree/95a5f5178fed90dc193c5c934f4a61559f203cf8/boards/arm/nrf5340dk_nrf5340) so I renamed the files to that. Should work with that as well since the nRF7002DK has the nrf5340 as well. Added a comment here: https://github.com/zephyrproject-rtos/zephyr/pull/64823/files#diff-8971d1014b6cb1e62fc6e2b5e679f34fc2e9b7b15d07f26abf385c5d034e8698R105-R106

Reply:
I tested this on nrf5340dk yesterday and can confirm it works